localisation/00_wc_artifact_localisation.csv

```diff
@@ -65,10 +65,10 @@ twilight_canticle;Twilight Canticle;;Loblied des Zwielichts;;;;;;;;;;;x
 xalatath;Xal'atath, Blade of the Black Empire;Xal'atath, lame de l'Empire noir;Xal'atath, Klinge des schwarzen Imperiums;;;;;;;;;;;x
 xalatath_desc;Nobody knows exactly when this blade was created, but according to legend it was used for sacrifices in ancient times. It drips with void magic and its quiet whispering attacks those with weak minds.;;Niemand wei� genau, wann diese Klinge geschaffen wurde, aber den Sagen zufolge wurde sie offenbar in alten Zeiten f�r Opferungen verwendet. Sie trieft vor Leeren-Magie und ihr leises Fl�stern bef�llt diejenigen mit einem schwachen Verstand.;;;;;;;;;;;x
-twin_blades_of_azzinoth;Twin Blades of Azzinoth;;Kriegsgleven von Azzinoth;;;;;;;;;;;
-twin_blades_of_azzinoth_desc;These twin-blades formerly belonged to Azzinoth, the leader of the doomguard, who was slain by Illidan 10.000 years ago. They are usually connected, but can be seperated and used as two weapons.;;Diese Doppelklingen geh�rten einst Azzinoth, dem Anf�hrer der Schreckenswache, der vor 10.000 Jahren von Illidan get�tet wurde. Normalerweise sind sie verbunden, k�nnen aber getrennt und als zwei Waffen benutzt werden.;;;;;;;;;;;
-gorehowl;Gorehowl;;Blutschrei;;;;;;;;;;;
-gorehowl_desc;Massive axe wielded by Grommash Hellscream of the Warsong Clan. It is permanently stained with the blood of the warrior's fallen enemies. Small holes and notches that dot the blade make it "sing" when swung.;;Eine riesige von Grommash Hellscream des Warsong-Clans benutzte Axt. Sie ist permanent mit dem Blut gefallener Opfer des Kriegers beschmiert. Kleine L�cher und Einkerbungen an der Klinge lassen die Axt �singen� wenn sie geschwungen wird.;;;;;;;;;;;
+twin_blades_of_azzinoth;Twin Blades of Azzinoth;;Kriegsgleven von Azzinoth;;;;;;;;;;;x
+twin_blades_of_azzinoth_desc;These twin-blades formerly belonged to Azzinoth, the leader of the doomguard, who was slain by Illidan 10.000 years ago. They are usually connected, but can be seperated and used as two weapons.;;Diese Doppelklingen geh�rten einst Azzinoth, dem Anf�hrer der Schreckenswache, der vor 10.000 Jahren von Illidan get�tet wurde. Normalerweise sind sie verbunden, k�nnen aber getrennt und als zwei Waffen benutzt werden.;;;;;;;;;;;x
+gorehowl;Gorehowl;;Blutschrei;;;;;;;;;;;x
+gorehowl_desc;A massive axe wielded by Grommash Hellscream of the Warsong Clan. It is permanently stained with the blood of the warrior's fallen enemies. Small holes and notches that dot the blade make it "sing" when swung.;;Eine riesige von Grommash Hellscream des Warsong-Clans benutzte Axt. Sie ist permanent mit dem Blut gefallener Opfer des Kriegers besudelt. Kleine L�cher und Einkerbungen an der Klinge lassen die Axt �singen� wenn sie geschwungen wird.;;;;;;;;;;;x
```

Comment:
Changed beginning segment to match Twin Blades above
```suggestion
gorehowl_desc;This massive axe was originally wielded by Grommash Hellscream of the Warsong Clan. It is permanently stained with the blood of the wielder's fallen enemies. Small holes and notches that dot the blade make it sound as if it is "singing" when the weapon is swung.;;Eine riesige von Grommash Hellscream des Warsong-Clans benutzte Axt. Sie ist permanent mit dem Blut gefallener Opfer des Kriegers besudelt. Kleine L�cher und Einkerbungen an der Klinge lassen die Axt �singen� wenn sie geschwungen wird.;;;;;;;;;;;x
```

Reply:
all artifact localisation was switched to frozen artifacts.